nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RRecipeSupport.groovy

```diff
@@ -110,16 +110,27 @@ abstract class RRecipeSupport
   }
 
   /**
-   * Matcher for packages mapping.
+   * Matcher for packages.gz mapping.
    */
-  static Builder packagesMatcher() {
+  static Builder packagesGzMatcher() {
```

Comment:
can we do here as we have for YUM? https://github.com/sonatype/nexus-internal/blob/bd3ff99c3b28f6d21b1d8b930f66e50d2440ba06/private/plugins/nexus-repository-yum/src/main/java/org/sonatype/nexus/repository/yum/internal/YumRecipeSupport.groovy#L177

Reply:
I mean combine  new TokenMatcher('/{path:.+}/PACKAGES.gz') and  new TokenMatcher('/{path:.+}/{filename:.+}.rds')

Reply:
Nice idea, done in f5a0c4341c6ed3aa9669cb5cd973d162f75a6acd